From 151ce4cf88cf2ab647d14064140f516bd78209b2 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Sun, 19 Jul 2026 20:12:35 -0700
Subject: [PATCH 06003/15333] posix-timers: Clean up kernel-doc warnings

Use the struct keyword when describing a struct.
Add missing kernel-doc for @it_pid_type.

Warning: ./include/linux/posix-timers.h:58 cannot understand
 function prototype: 'struct cpu_timer'
Warning: ./include/linux/posix-timers.h:222 struct member
 'it_pid_type' not described in 'k_itimer'

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Thomas Gleixner <tglx@kernel.org>
Link: https://patch.msgid.link/20260720031236.3070575-1-rdunlap@infradead.org
---
 include/linux/posix-timers.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/linux/posix-timers.h b/include/linux/posix-timers.h
index 4d3dbcef379e..4117827efdb8 100644
--- a/include/linux/posix-timers.h
+++ b/include/linux/posix-timers.h
@@ -47,7 +47,7 @@ static inline bool clockid_aux_valid(clockid_t id)
 #include <linux/signal_types.h>
 
 /**
- * cpu_timer - Posix CPU timer representation for k_itimer
+ * struct cpu_timer - Posix CPU timer representation for k_itimer
  * @node:	timerqueue node to queue in the task/sig
  * @head:	timerqueue head on which this timer is queued
  * @pid:	Pointer to target task PID
@@ -174,6 +174,7 @@ static inline void posix_cputimers_init_work(void) { }
  * @it_sigqueue_seq:	The sequence count at the point where the signal was queued
  * @it_sigev_notify:	The notify word of sigevent struct for signal delivery
  * @it_interval:	The interval for periodic timers
+ * @it_pid_type:	The type of the PID
  * @it_signal:		Pointer to the creators signal struct
  * @it_pid:		The pid of the process/task targeted by the signal
  * @it_process:		The task to wakeup on clock_nanosleep (CPU timers)
-- 
2.55.0

